dev-requirements.txt

```diff
@@ -1,6 +1,6 @@
 pypiwin32
 six
-Pillow==3.0.0
+Pillow==4.0.0
```

Comment:
It's for Py3.6 compatibility. Doesn't work with Pillow==3.0.0 or even 3.4.2 (no wheels for py36).
Pillow==4.0.0 contains wheels for all pythons: 2.7 and 3.3+.